examples/google-esp-integration/with_sdk_v3/README.md

```diff
@@ -1,15 +1,14 @@
 # UID2 SDK ESP Integration Example
 
-This example demonstrates how a content publisher who are working with (Google Publisher Tags) can use the ESP (Google Encrypted Signals for Publishers) and the [Client-Side Identity JavaScript SDK](https://github.com/UnifiedID2/uid2docs/blob/main/api/v1/sdks/client-side-identity-v1.md) (also known as the UID2 SDK) to share UID2 direclty with bidders.
+This example demonstrates how a content publisher who is working with Google publisher tags can use the [Google Encrypted Signals program (ESP)](https://support.google.com/admanager/answer/10488752) and the [Client-Side Identity JavaScript SDK](https://github.com/UnifiedID2/uid2docs/blob/main/api/v1/sdks/client-side-identity-v1.md) (also known as the UID2 SDK) to share UID2 directly with bidders, in an implementation that uses this SDK.
```

Comment:
Sorry I forgot to update Google Encrypted Signals program (ESP) to Google secure signals